identity/app/views/profile/emailPrefs.scala.html

```diff
@@ -70,9 +70,20 @@ <h2 class="form__heading">Newsletter options</h2>
 
     </div>
 
-    <div class="manage-account__modal manage-account__modal--newsletterFormat">
+    <fieldset class="fieldset fieldset--manage-account-rightsubmit js-manage-account__ajaxForm-submit">
+        <div class="fieldset__heading"></div>
+        <div class="fieldset__fields">
+            <ul class="u-unstyled">
+                <li>
+                    <button type="submit" class="manage-account__button" data-link-name="Save email preferences">Save changes</button>
+                </li>
+            </ul>
+        </div>
+    </fieldset>
+
+    <div id="manage-account__modal--newsletterFormat" class="manage-account__modal manage-account__modal--newsletterFormat" role="dialog">
         <div class="manage-account__modalContent">
-            <button class="manage-account__modalCloser js-manage-account__modalCloser">
+            <button class="manage-account__modalCloser js-manage-account__modalCloser" alt="Close this window" title="Close this window">
```

Comment:
I'd say it's better to

```html
<button>
  <span class="visually-hidden">Close this window</span>
  {{icon}}
</button>
```

What do you think?

Reply:
wow yeah just read on this and `title` is kinda useless innit